lib/api-call.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
consider making this into a separate node package, or at least keeping it decoupled (as you have done) from the rendering side of things so that you can do that.